Userland/Applications/CrashReporter/main.cpp

```diff
@@ -41,19 +41,30 @@ struct TitleAndText {
 static NonnullRefPtr<GUI::Window> create_progress_window()
 {
     auto window = GUI::Window::construct();
+
     window->set_title("CrashReporter");
     window->set_resizable(false);
     window->resize(240, 64);
     window->center_on_screen();
+
+    window->on_close = [&]() {
+        auto& progressbar = *window->main_widget()->find_descendant_of_type_named<GUI::Progressbar>("progressbar");
+        if (progressbar.value() != progressbar.max())
+            exit(0);
+    };
```

Comment:
If you move this block of code below the creation of `progressbar`, you can use `progressbar` directly in the lambda instead of having to `find_descendant_of_type_named` :)